big_tests/tests/mod_global_distrib_SUITE.erl

```diff
@@ -313,6 +341,9 @@ generic_end_per_testcase(CaseName, Config) ->
       get_hosts()),
     escalus:end_per_testcase(CaseName, Config).
 
+virtual_hosts() ->
+    [<<"localhost">>, <<"localhost.bis">>].
```

Comment:
Can we take this values from `test.config`? It would be 
```suggestion
    [ct:get_config({hosts, mim, domain}), ct:get_config({hosts, mim, secondary_domain})].
```